app/assets/javascripts/admin/addon/components/admin-config-areas/api-keys-new.gjs

```diff
@@ -123,6 +127,21 @@ export default class AdminConfigAreasApiKeysNew extends Component {
     return enabledScopes.flat();
   }
 
+  @bind
+  atLeastOneGranularScope(data, { addError, removeError }) {
+    removeError("scopes");
```

Comment:
Needed to clear the error before the second try in the case where the user has seen the message and selected some scopes.